### DISCUSSION TITLE
Nikolaos Dimopoulos Phalcon Developer

### DISCUSSION THREAD

**Discussion by ghost** [phalcon/phalcon#87] [General]
I was concerned about Phalcon v5 been developed mostly by one programmer "Niden" and not many others contributing. But than I found Niden's blog @ https://niden.net/  and read few posts. Now I know Phalcon is in the good hands and will have a great future. Thank you for stepping up Niden, you seem to be very skilled programmer. I have no doughts of using Phalcon in the future.

**Comment by niden** [comment]
I might be the only one for the moment the contributes to v5 but there are others doing other stuff as well. We also support v4 for now, documents, incubator and many more. More developers will be added in the list of contributors for v5 as time goes on.
